models/build.js

```diff
@@ -92,10 +92,6 @@ const MODEL = {
         .default({})
         .description('Key=>Value information from the build itself'),
 
-    steps: Joi
```

Comment:
Can we remove `steps` from model? I'm worried this makes validation failure like that line.
https://github.com/screwdriver-cd/screwdriver/blob/b2aa7c77fdd4470e87e172592489575009151ea6/plugins/builds/get.js#L51

Reply:
I see.
this fixed.